fixtures/eurolite/led-bar-3-hcl-bar.json

```diff
@@ -0,0 +1,303 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Bar-3 HCL Bar",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Rainer Jung"],
+    "createDate": "2020-01-13",
+    "lastModifyDate": "2020-01-13"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.de/download/51930395-Manual-110557-1.000-eurolite-led-bar-3-hcl-bar-de_en.pdf"
+    ],
+    "productPage": [
+      "https://www.steinigke.de/en/mpn51930395-eurolite-led-bar-3-hcl-bar.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=SV2AFTdJKNE"
+    ]
+  },
+  "physical": {
+    "dimensions": [265, 129, 87],
+    "weight": 1.13,
+    "power": 12,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [20, 20]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Amber": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "UV": {
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Mode": {
```

Comment:
Could you please use [`effectPreset`](https://github.com/OpenLightingProject/open-fixture-library/blob/master/docs/capability-types.md#effect) instead of `effectName` in this channel's `Effect` capabilities?

Reply:
I think, I missunderstood this.
![image](https://user-images.githubusercontent.com/11313/72362812-188c7600-36f4-11ea-9fed-515c6f1a3e13.png)

Reply:
Ah, reading the documentation helps.

Reply:
Sorry, my request was unclear: The fade capabilities should have `"effectPreset": "ColorFade"` or `"effectPreset": "ColorJump"` instead of the free text `effectName`. Together with the `speedStart` / `speedEnd` / `soundControlled` properties, it defines everything included in the free text in a machine-readable fashion.